tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/konan/KonanPlugin.kt

```diff
@@ -206,6 +206,7 @@ internal fun MutableList<String>.addFileArgs(parameter: String, values: Collecti
 internal fun MutableList<String>.addListArg(parameter: String, values: List<String>) {
     if (values.isNotEmpty()) {
         addArg(parameter, values.joinToString(separator = " "))
+        println("$parameter, ${values.map { "\"$it\"" }.joinToString(separator = " ")}")
```

Comment:
Debug output?

Reply:
Yes, thank you. This function now isn't used at all. Is it better to remove it?

Reply:
Yes, why not?